controllers/etcd_controller.go

```diff
@@ -1066,23 +1137,29 @@ func canDeleteStatefulset(sts *appsv1.StatefulSet, etcd *druidv1alpha1.Etcd) boo
 
 }
 
-func (r *EtcdReconciler) updateEtcdErrorStatus(etcd *druidv1alpha1.Etcd, sts *appsv1.StatefulSet, lastError error) error {
+func (r *EtcdReconciler) updateEtcdErrorStatus(etcd *druidv1alpha1.Etcd, sts *appsv1.StatefulSet, lastError error, description string, operationType druidv1alpha1.LastOperationType) error {
```

Comment:
Are there multiple possible values for `operationType`? If no, can be bake it into this func and avoid the extra parameter?